keyboards/alps64/config.h

```diff
@@ -48,6 +48,10 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* Locking resynchronize hack */
 #define LOCKING_RESYNC_ENABLE
 
+/* Bootmagic Lite Support*/
+#define BOOTMAGIC_LITE_ROW 3
+#define BOOTMAGIC_LITE_COLUMN 6
```

Comment:
I've replaced the default Bootmagic Lite row and column of 0,0 (first switch to the right of the spacebar, right alt in my layout) to 3,6 which is on the top left of the keyboard right under the usb port (that's tilde on my layout).